jina/peapods/runtimes/container/__init__.py

```diff
@@ -29,6 +33,23 @@ def __init__(self, args: 'argparse.Namespace', ctrl_addr: str):
                 'the container fails to start, check the arguments or entrypoint'
             )
 
+    @property
+    def _host_ctrl(self) -> str:
+        """
+        Checks if caller (JinaD) has set `docker_kwargs['extra_hosts']` to _docker_host.
+        If yes, set host_ctrl to _docker_host, else set it to localhost
+
+        :return: host for control port
+        """
+        _docker_host = 'host.docker.internal'
```

Comment:
I wonder if this should be some constant somewhere in the code, that can be used at several places!?

Reply:
It is used as a constant (`__docker_host__`) in `daemon/__init__.py`, we might have to move it to `jina/__init__.py`